src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -174,15 +178,15 @@ public Task Initialize(TimeSpan timeout)
             TimeSpan oldest = dequeueTimeUtc - messages[0].EnqueuedTimeUtc;
             TimeSpan newest = dequeueTimeUtc - messages[messages.Count - 1].EnqueuedTimeUtc;
 #endif
-            monitor.TrackAgeOfMessagesRead(oldest, newest);
+            monitor?.TrackMessagesReceived(messages.Count, oldest, newest);
```

Comment:
Since we're changing this, let's report cleaner information.   We originally reported the age, but that's suspect and the monitor can always calculate that if needed from the enqueue times.  Let's report the enqueue times rather than the relative ages.